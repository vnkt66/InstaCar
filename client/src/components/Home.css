body {
    background: radial-gradient(circle, rgba(238,174,202,1) 0%, rgba(148,187,233,1) 100%);
}
.carlogo {
 margin: 15px 0;
}
.support {
   position: relative;
   /* border: 1px solid red; */
}
#supporticon {
    position: absolute;
    top: 27px;
    left: 50px;
}
#supportort {
    position: absolute;
    margin-left: 40px;
    left: 44px;
    top: 20px;
    font-size: 12px;
    color: black;
}
#supportnumber {
    position: absolute;
    top: 36px;
    left: 84px;
    font-size: 15px;
}
.top1 {
   margin-top: 120px;   
}
#top1content {
    height: 80px;
    text-align: center;
}
#top1contenttext {
    font-size: 35px;
    letter-spacing: 5px;
}