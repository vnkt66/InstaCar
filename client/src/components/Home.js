import React, { Component } from 'react';
// import { Link } from 'react-router-dom';
import { Container, Grid, Icon, Segment } from 'semantic-ui-react';
import './Home.css';
import ContainerTab from './TabPanel';

class Home extends Component {
    render() {
        return (
          <div className="Home">
              <Container fluid className="top">
              <Grid columns='equal'>
              <Grid.Column className="carlogo">
               <a href="/aq-index" id="carlogoimg">
               <img src="https://instacar.in/wp-content/themes/instacar/images/Insta-car-01-2.png" alt="instacar-logo" />
               </a>
              </Grid.Column>
              <Grid.Column width={8}>
                  
              </Grid.Column>
              <Grid.Column className="support">
               <a href="tel:+91-9902202299">
               <Icon inverted id="supporticon" name='phone' size='large' color="grey"/>
               </a>
               <span id="supportort" style={{color: 'white', fontWeight: 'bolder'}}>24x7 Support</span>
               <span id="supportnumber" style={{color: 'white', fontWeight: 'bolder'}}>+91-9902202299</span>
              </Grid.Column>
              </Grid>
              </Container>
              <Container fluid className="top1">
              <Grid id="top1content" style={{marginBottom: '30px'}}>
                <Grid.Column width={16}>
                <h1 id="top1contenttext" style={{color: 'white'}}>Chauffeur Driven, On Demand.</h1>
                </Grid.Column>
              </Grid>
              <Container>
              <Grid>
                <Segment style={{width: '100%', background: 'border-box'}}>
                  <ContainerTab />
                </Segment>
              </Grid>
              </Container>
              </Container>
            </div>
          );
    }
}

export default Home;
