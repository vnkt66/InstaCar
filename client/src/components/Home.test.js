import React from 'react';
import { configure, shallow } from 'enzyme';
import Adapter from 'enzyme-adapter-react-16';

import Home from './Home';

import ContainerTab from './TabPanel';
import DatePicker from "react-datepicker";

configure({ adapter: new Adapter() });

describe('Home component Shallow Rendering', () => {
    it('renders without crashing', () => {
       const component = shallow(<Home />);

       expect(component).toMatchSnapshot();
     });
 });

 describe('ContainerTab', () => {
    let wrapper;
  
    it('wraps class', () => {
      wrapper = shallow(<ContainerTab />);
      expect(wrapper.find(DatePicker));
    });
});



