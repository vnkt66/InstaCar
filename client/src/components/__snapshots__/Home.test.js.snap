// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Home component Shallow Rendering renders without crashing 1`] = `ShallowWrapper {}`;
